components/infobox/commons/infobox_player.lua

```diff
@@ -0,0 +1,227 @@
+local Class = require('Module:Class')
+local Cell = require('Module:Infobox/Cell')
+local Infobox = require('Module:Infobox')
+local Template = require('Module:Template')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+local Namespace = require('Module:Namespace')
+local Links = require('Module:Links')
+local Flags = require('Module:Flags')
+--local Lua = require('Module:Lua')--needed for the next line only
+--local AgeCalculation = (Lua.requireIfExists('Module:Infobox/Player/Age') or require('Module:Infobox player/age')).get
+
+local getArgs = require('Module:Arguments').getArgs
+
+local Player = Class.new()
+local Language = mw.language.new('en')
+local _LINK_VARIANT = 'player'
+
+function Player.run(frame)
+    return Player:createInfobox(frame)
+end
+
+function Player:createInfobox(frame)
+    local args = getArgs(frame)
+    self.frame = frame
+    self.pagename = mw.title.getCurrentTitle().text
+    self.name = args.id or self.pagename
+
+    if args.game == nil then
+        return error('Please provide a game!')
+    end
+
+    local infobox = Infobox:create(frame, args.game)
+
+    local earnings = Player:calculateEarnings(args)
+    Variables.varDefine('earnings', earnings)
+    if earnings == 0 then
+        earnings = nil
+    else
+        earnings = '$' .. Language:formatNum(earnings)
+    end
+    local birthDisplay, deathDisplay, birthday, deathday = Player:birthAndDeath(args)
+    local status = Player:getStatus(args)
+
+    infobox :name(Player:name(args))
+            :image(args.image, args.default)
+            :centeredCell(args.caption)
+            :header('Player Information', true)
+            :cell('Name', args.name)
+            :cell('Romanized Name', args.romanized_name)
+            :cell('Birth', birthDisplay)
+            :cell('Died', deathDisplay)
+            :cell('Status', status)
```

Comment:
```suggestion
            :cell('Status', Player:getStatus(args))
```

Reply:
it gets stored in lpdb, hence i set it as a local above, because we need it there again
maybe will change it to a table `status = {display=..., store=...}` (returned via the function)

Reply:
Ah, alright then